ajax/libs/vue-lazyload/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vue-lazyload",
+  "filename": "vue-lazyload.js",
+  "version": "1.1.2",
```

Comment:
So it's `1.1.2` or `1.0.6`?

Reply:
It's v1.0.6. I did really need to be more careful as you said.